aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -263,30 +263,24 @@ When using the out-of-process hosting model, the app might not shut down immedia
-
-![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-inproc.png)
-![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-outofproc.png)
+::: moniker range=">= aspnetcore-2.2"
 
-For inprocess, if the ASP.NET Core module fails to start the application, a *500.30 Start Failure* status code page appears.
+Both in-process and out-of-process hosting produce custom error pages when they fail to start the app.
 
-![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500_30-inproc.png)
+If the ASP.NET Core Module fails to find either the in-process or out-of-process request handler, a *500.0 - In-Process/Out-Of-Process Handler Load Failure* status code page appears.
 
-For out of process, if the ASP.NET Core Module fails to launch the backend process or the backend process starts but fails to listen on the configured port, a *502.5 Process Failure* status code page appears.
+For in-process hosting if the ASP.NET Core Module fails to start the app, a *500.30 - Start Failure* status code page appears.
 
-![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-502_5-outofproc.png)
+For out-of-process hosting if the ASP.NET Core Module fails to launch the backend process or the backend process starts but fails to listen on the configured port, a *502.5 - Process Failure* status code page appears.
 
-To suppress this page and revert to the default IIS 5xx status code page, use the `disableStartUpErrorPage` attribute. For more information on configuring custom error messages,  [HTTP Errors `<httpErrors>`](/iis/configuration/system.webServer/httpErrors/).
+To suppress this page and revert to the default IIS 5xx status code page, use the `disableStartUpErrorPage` attribute. For more information on configuring custom error messages, see [HTTP Errors &lt;httpErrors&gt;](/iis/configuration/system.webServer/httpErrors/).
 
 ::: moniker-end
 
 ::: moniker range="< aspnetcore-2.2"
 
-If the ASP.NET Core Module fails to launch the backend process or the backend process starts but fails to listen on the configured port, a *502.5 Process Failure* status code page appears. To suppress this page and revert to the default IIS 502 status code page, use the `disableStartUpErrorPage` attribute. For more information on configuring custom error messages,  see [HTTP Errors `<httpErrors>`](/iis/configuration/system.webServer/httpErrors/).
+If the ASP.NET Core Module fails to launch the backend process or the backend process starts but fails to listen on the configured port, a *502.5 - Process Failure* status code page appears. To suppress this page and revert to the default IIS 502 status code page, use the `disableStartUpErrorPage` attribute. For more information on configuring custom error messages, see [HTTP Errors &gt;httpErrors&gt;](/iis/configuration/system.webServer/httpErrors/).
```

Comment:
Two &gt; symbols

Reply:
Clarification? You prefer `\<httpErrors>`?

Reply:
no it's just that both open and end characters are the same, open should be &lt;, not &gt;

Reply:
DAH ... my bad.

> \[HTTP Errors &gt;httpErrors&gt;]

... so much for "cut-'n-paste"